spec/models/post_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  user = User.new(name: 'name', photo: 'photo', bio: 'bio', posts_counter: 0)
+  subject { Post.new(author: user, title: 'title', text: 'text', comments_counter: 0, likes_counter: 0) }
```

Comment:
Great Job, The test you've written to check that a zero `posts_counter` is valid is spot on, and this is critical for preventing invalid data from entering your system ✔️ 💪 💪 💪